tests/test_codecs.py

```diff
@@ -134,10 +130,9 @@ def test_context_to_readable_headers(self):
                     'trace-id': '100:7f:0:1',
                     'trace-attr-bender': 'Countess%20de%20la%20Roca',
                     'trace-attr-fry': 'Leela',
-                    'trace-attr-key1': '%FF',
-                    'trace-attr-key2-caf\xc3\xa9': 'caf%C3%A9',
-                    'trace-attr-key3': 'caf%C3%A9',
-                    'trace-attr-key4-caf\xc3\xa9': 'value',
```

Comment:
isn't this still a useful testcase?

Reply:
updated the PR description: This change **officially removes support for non-ascii characters in baggage keys.**